src/pancam.py

```diff
@@ -0,0 +1,22 @@
+from follower import Follower
+
+follower = Follower()
+
+def seek():
+    #tag_data = follower.detect()
+    turn = follower.turn()
+
+    if follower.detect() == False:
```

Comment:
It's more Pythonic (and goes well with the rest of the code) to do:
`if not follower.detect()`